src/modules/evbc/rpc/websocketRpcIssuer.ts

```diff
@@ -14,7 +14,7 @@ export class WebsocketRpcIssuer extends RpcIssuer {
     private publish_connection_state: (status: ConnectionStatus) => void,
   ) {
     super();
-    this._socket = new WebSocket(this._url);
+    this._socket = new WebSocket(this._url, "everest-controller");
```

Comment:
It would be nice to add a comment above explaining why this is necessary, as this feels pretty non-standard.

Also this likely has to be adapted here as well: https://github.com/EVerest/everest-admin-panel/blob/7fd968d21a76ee8b484e4fdc9a69bd3f25dbc2d1/src/modules/evbc/simulator-sample-data/update-simulator-data.js#L36